crates/assembly/src/instruction/mod.rs

```diff
@@ -444,6 +444,28 @@ impl Assembler {
                     instruction.span(),
                 )?
             },
+            Instruction::LocLoadWLe(v) => {
+                let local_addr = v.expect_value();
+                if !local_addr.is_multiple_of(WORD_SIZE as u16) {
+                    return Err(RelatedLabel::error("invalid local word index")
+                        .with_help("the index to a local word must be a multiple of 4")
+                        .with_labeled_span(v.span(), "this index is not word-aligned")
+                        .with_source_file(
+                            proc_ctx.source_manager().get(proc_ctx.span().source_id()).ok(),
+                        )
+                        .into());
+                }
+
+                mem_ops::mem_read(
+                    block_builder,
+                    proc_ctx,
+                    Some(local_addr as u32),
+                    true,
+                    false,
+                    instruction.span(),
+                )?;
```

Comment:
I would pull this out in a function (instead of copy-pasting), and then the relationship with `_be()` becomes clear: `mem_read()`, followed by `reversew`